project.clj

```diff
@@ -1,11 +1,11 @@
-(defproject ring/ring-mock "0.4.0"
+(defproject ring/ring-mock "0.5.0"
```

Comment:
Version changes should not be in the PR, as per the contributing guidelines.

Reply:
Reverted version back to 0.4.0